iidm/iidm-extensions/src/main/java/com/powsybl/iidm/network/extensions/DiscreteMeasurement.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.extensions;
+
+import java.util.Set;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+public interface DiscreteMeasurement {
+
+    enum Type {
+        TAP_POSITION,
+        SWITCH_POSITION,
+        SHUNT_COMPENSATOR_SECTION,
+        OTHER
+    }
+
+    enum TapChanger {
+        RATIO_TAP_CHANGER,
+        PHASE_TAP_CHANGER,
+        RATIO_TAP_CHANGER_1,
+        PHASE_TAP_CHANGER_1,
+        RATIO_TAP_CHANGER_2,
+        PHASE_TAP_CHANGER_2,
+        RATIO_TAP_CHANGER_3,
+        PHASE_TAP_CHANGER_3
+    }
+
+    String getId();
```

Comment:
I don't think we should enforce a non null ID, I only ever saw identified measurement in CGMES, it is not mandatory in other formats and it would be weird to invent some for IIDM